README

```diff
@@ -1 +1,24 @@
-Juexin's pull request test
+Tutorial of Running this project (Beta)
```

Comment:
In github, the main readme file is usually intended to give a general description for the whole project. However, contents of this file is only specific to one standalone script. It might be better to document individual script/modules in the github wiki instead of the main readme file.